typescript.js

```diff
@@ -41,7 +40,8 @@ module.exports = {
     eqeqeq: ['error', 'smart'],
     'new-cap': 'error',
     'no-extend-native': 'error',
-    'no-use-before-define': ['error', 'nofunc'],
+    'no-use-before-define': 'off',
```

Comment:
I think that we can simply remove this line as `@typescript-eslint/no-use-before-define`  extends vanilla 'no-use-before-define': https://typescript-eslint.io/rules/no-use-before-define, isn't it?